### ISSUE TITLE
[Feature request] 添加油猴脚本

### ISSUE THREAD

**Issue by fangyuan99** [Evil0ctal/Douyin_TikTok_Download_API#196]
**您的功能请求是否与问题相关？ 如有，请描述。**

我在使用tiktok和抖音的时候，如果有个按钮可以一键解析就好

**描述您想要的解决方案**

添加一个下载按钮，调用下载api，下载即可

**Comment by Evil0ctal** [comment]
iOS平台已经有一个快捷指令可以实现了，安卓我目前不知道有没有类似的东西。

**Comment by fangyuan99** [comment]
> iOS平台已经有一个快捷指令可以实现了，安卓我目前不知道有没有类似的东西。

我有尝试自己写了一个脚本，但是会被安全限制

```
Access to fetch at 'https://api.douyin.wtf/tiktok_video_data/?tiktok_video_url=https://www.tiktok.com/@bikini_collect/video/7189419900841069851?q=sexy%20girl&t=1681292271351' from origin 'https://www.tiktok.com' 
has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

```

**Comment by Evil0ctal** [comment]
可以搞一个浏览器插件

**Comment by fangyuan99** [comment]
> 可以搞一个浏览器插件

我写了一个插件版本的：
[fangyuan99/Tiktok-Downloader: 抖音 tiktok 无水印下载插件 Chrome Edge 都可以用](https://github.com/fangyuan99/Tiktok-Downloader)

**Comment by Evil0ctal** [comment]
Nice！你可以在这个仓库里搞一个文件夹，然后把插件放进去，如果你有时间的话😂

**Comment by fangyuan99** [comment]
> Nice！你可以在这个仓库里搞一个文件夹，然后把插件放进去，如果你有时间的话😂

我提交pr了，第一次弄，你看看哪不规范我再改。
[Add chrome plugin and tampermonkey script by fangyuan99 · Pull Request #198 · Evil0ctal/Douyin_TikTok_Download_API](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/pull/198)

**Comment by fangyuan99** [comment]
请问你是否忘记处理pr了

**Comment by Evil0ctal** [comment]
Closing as part of a cleanup for **v5.0.0**, released today.

v5 is a rewrite from an empty branch: different endpoints, different configuration, its own database schema, and none of v4's code. That means this issue was filed against a codebase that is no longer the one being developed, and leaving it open would imply someone is going to fix it there. Nobody is — v4 is frozen.

**If this still affects you:**

- **On v5** — please open a new issue. It will get a real answer, and it starts from a codebase where a lot of the old failure modes are gone: authentication on every endpoint, encrypted credentials, an identity pool that maintains itself, and a request log that shows what actually happened. The [Troubleshooting guide](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/blob/main/documents/en/14-troubleshooting.md) and `dtk diagnose` will answer most of what a maintainer would otherwise have to ask you.
- **Staying on v4** — it still works. The code is on the [`v4` branch](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/tree/v4) and the image stays on Docker Hub as `evil0ctal/douyin_tiktok_download_api:V4.1.2`. Pin that tag rather than `latest`; `latest` is v5 now.

Sorry to anyone whose report sat here without an answer. Closing 143 issues in one go is not a verdict on any of them — it is an admission that they were never going to get one on v4, and that saying so is better than leaving them open indefinitely.

→ [v5.0.0 release notes](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/releases/tag/v5.0.0)

---

作为 **v5.0.0** 发布（就在今天）的一次清理而关闭。

v5 是从空分支重写的：接口不同、配置方式不同、数据库结构是自己的一套，且不包含 v4 的任何代码。也就是说，这条 issue 针对的代码库已经不是在维护的那个了，继续开着会让人以为还有人会在那边修它——没有了，v4 已冻结。

**如果这个问题对你仍然存在：**

- **在 v5 上** —— 请新开一条 issue，会得到真正的回复。而且它面对的是一个大部分老问题已经不复存在的代码库：每个端点都要认证、凭据加密存储、身份池会自己维护、请求日志能看到实际发生了什么。[故障排查文档](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/blob/main/documents/zh/14-troubleshooting.md)和 `dtk diagnose` 能回答维护者原本要反过来问你的大部分问题。
- **继续用 v4** —— 它照常能跑。代码在 [`v4` 分支](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/tree/v4)，镜像以 `evil0ctal/douyin_tiktok_download_api:V4.1.2` 保留。请固定这个版本号 tag，别用 `latest`——`latest` 现在是 v5 了。

对所有报告一直没等到回复的人说声抱歉。一次性关掉 143 条不是对其中任何一条的评判，而是承认它们在 v4 上本来就等不到回复了，而把这件事说清楚比让它们无限期开着要好。

→ [v5.0.0 更新说明](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/releases/tag/v5.0.0)
